### ISSUE TITLE
(suggestion) .b timer

### ISSUE THREAD

**Issue by MrDodojo** [cabaletta/baritone#461]
Have a timer that cancels your action what you are doing after an amount of time you enter

**Comment by ZeroMemes** [comment]
If this were added I propose it be added onto the existing `cancel` command, where there could be an optional parameter specifying the amount of time: `cancel 5s` for example.

**Comment by MrDodojo** [comment]
That's an even better implementation! Didn't think of that.

**Comment by TypicalHog** [comment]
This would be awesome to have. There should also be a command to cancel the action when there is a certain amount of a specific item in an inventory. Something like cancelWhenHave coal_ore 48 for example.

**Comment by MrDodojo** [comment]
I don't think WhenHave is really fitting, what about .b cancel ItemAmount coal_ore 48?
And the timed one would be .b cancel time 1h

**Comment by TypicalHog** [comment]
> I don't think WhenHave is really fitting, what about .b cancel ItemAmount coal_ore 48?
> And the timed one would be .b cancel time 1h

I agree 100%, we could also have .b cancel hp <8 or .b cancel hunger <6 or .b cancel tool <250 etc.
